kart/raster/v1.py

```diff
@@ -26,3 +32,103 @@ def set_tile_extension(cls, filename, ext=None, tile_format=None):
     @classmethod
     def extract_tile_metadata_from_filesystem_path(cls, path):
         return extract_raster_tile_metadata(path)
+
+    @classmethod
+    def get_tile_path_pattern(
+        cls, tilename=None, *, parent_path=None, include_conflict_versions=False
+    ):
+        return get_tile_path_pattern(
+            tilename,
+            parent_path=parent_path,
+            include_conflict_versions=include_conflict_versions,
+        )
+
+    def diff_to_working_copy(
+        self,
+        workdir_diff_cache,
+        ds_filter=DatasetKeyFilter.MATCH_ALL,
+        *,
+        convert_to_dataset_format=False,
+        skip_gdalinfo=False,
```

Comment:
minor: positive flags are easier to reason about (`use_gdalinfo=True`)

Reply:
Renamed to `extract_metadata`, and the same for point-clouds.
Possibly, this fixed some functionality for rasters that I haven't yet looked into involving kart reset. In further PRs I will need to add more tests to make sure rasters is at feature parity with point-clouds and find out where it isn't